README.md

```diff
@@ -35,4 +35,5 @@ In this project, you will be coding a mortgage calculator. At this point, you sh
 
 ![Git with Android Studio Documentation](Git_with_AndroidStudio.md)
 
-
```

Comment:
Should this go above the "Android Project Setup with Android Studio" section since it's the first thing they'll have to do? Also, maybe we should add something specifying that it only needs to be done once, and can be done by one of the lead devs? 

Last thing, should we add a card about this to the trello board? maybe create a " Week 2 Lead Dev: Coding Setup"